acsoo/tools.py

```diff
@@ -96,3 +97,16 @@ def _find_executable(exe):
 
 def cfg_path(filename):
     return os.path.join(os.path.dirname(__file__), 'cfg', filename)
+
+
+def parse_requirements(requirements_string):
+    """
+    This method parses the given requirements string into a dictionary sorted
+    by name.
+    :param requirements_string: requirements string
+    :return: Dictionary like : {'addon_name': Requirement object}
+    """
+    parsed_requirements = {}
+    for req in requirements.parse(requirements_string):
+        parsed_requirements[req.name] = req
+    return parsed_requirements
```

Comment:
I tend to think this utility method has little added value compared to

`{req.name: req for req in requirements.parse(requirements_string)}`